application/web/scripts/code-mirror-script.js

```diff
@@ -0,0 +1,34 @@
+// LRC
+//////////////////////////////////////////////////////////////////////////////////
+///* This is where the CodeMirror editor is applied to the entry-textarea div *///
+//////////////////////////////////////////////////////////////////////////////////
+
+//TODO: 
+// * Full programming language support
+// * Live rendering 
+// * Modify styling
+
+document.addEventListener("DOMContentLoaded", function() {
+    
+    var textArea = document.getElementById("entry-textarea")
+    
+    // Create the CodeMirror Object
+    var editor = CodeMirror(textArea, {
```

Comment:
apparently there's a lint and test check that failed because "Code Mirror" is not defined. Also could you use "let" instead of var for this line and line 13

Reply:
yeah let me change it, and I'm confused by that as well, is it something that needs to be included in the package.json?

Reply:
I'm not sure maybe? Might be a Cameron question haha

Reply:
lmao true